app/assets/stylesheets/mtl/extend/_avatars.scss

```diff
@@ -1,24 +1,26 @@
 // avatars
 .avatar,
 .mtl-layout-default .side-nav .avatar {
-  display: block;
-  width: 52px;
-  height: 52px;
-  line-height: 52px;
-  padding: 0;
-  border-radius: 26px;
+  display: inline-block;
+  border-radius: 50%;
+  height: 50px;
```

Comment:
@lwe: found so far: 52x52, 32x32, 25x25... so yeah, we could go for .avatar-small (25), .avatar-large (52). what do you think?